hedera-node/src/main/java/com/hedera/services/state/exports/SignedStateBalancesExporter.java

```diff
@@ -109,25 +105,36 @@ public SignedStateBalancesExporter(
 		this.signer = signer;
 		this.expectedFloat = properties.getLongProperty("ledger.totalTinyBarFloat");
 		this.dynamicProperties = dynamicProperties;
+		exportPeriod = dynamicProperties.balancesExportPeriodSecs();
+	}
+
+	private Instant getFirstExportTime(Instant now, final int exportPeriodInSecs) {
+		final long epochSeconds = now.getEpochSecond();
+		long remainingSecs = epochSeconds % exportPeriodInSecs;
+		if(remainingSecs == 0) {
+			return Instant.ofEpochSecond(now.getEpochSecond());
+		}
+		else {
+			return Instant.ofEpochSecond(now.plusSeconds(exportPeriodInSecs - remainingSecs).getEpochSecond());
```

Comment:
The `if-else` logic could be reduced to 
`return Instant.ofEpochSecond(now.plusSeconds(exportPeriodInSecs - remainingSecs).getEpochSecond())`;
`elapsedSecs` looks like a better term compared to `remainingSecs`.

Reply:
Agree.  renamed and refactored with ternary operator. The suggested reducing changed the original logic.